src/library/baseexternalplaylistmodel.h

```diff
@@ -28,11 +30,19 @@ class BaseExternalPlaylistModel : public BaseSqlTableModel {
     bool isColumnInternal(int column) override;
     Qt::ItemFlags flags(const QModelIndex &index) const override;
     CapabilitiesFlags getCapabilities() const override;
+    QString modelKey() const override {
+        return pointerToQString(this) %
```

Comment:
Please use '+' instead of '%' as stated in the Coding Guidelines

> From Mixxx 2.3 we set QT_USE_QSTRINGBUILDER to use QStringBuilder for operator+. Use + in favour of % for better readability.